packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -83,6 +82,7 @@ export const App = ({ flowQueryParams, navigatorLanguages }: AppProps) => {
               to="/settings#connected-services"
               noThrow
             />
+            <Redirect from="/avatar/change" to="/settings/avatar/" noThrow />
```

Comment:
You could probably leave a comment here for future readers why this exists, Sync settings in Firefox link directly to the avatar page.